strapi/strapi
# Fix Content History: Crash when viewing versions with added media or relation fields

### What does it do?

It fixes the error that we had when you opened the History of a collection where you added some new fields (media field or relation field) after the creation of an entry and click on the versions before the new fields were added and after you received an error

### Why is it needed?

To avoid the app crashing if you switch between the versions

### How to test it?

1. Create a new collection with some fields on it
2. then create an entry of this collection and save
3. go back to the Content Type Builder and add some new fields to the collection (one image or one relation or both)
4. Update the entry previously created with some values for the new fields and save
5. Go to the Content History of the Document and click the different version from one without the new fields and one with the new fields
6. You don't have to see the error

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/20616
